addons/README.md

```diff
@@ -5,8 +5,6 @@ how to use the KubeOne Addons, consider the [KubeOne documentation][addons-docs]
 
 ## Available Addons
 
-* [Cluster backups (with Restic)][backups-addon]
+* [Cluster backups (with Restic)](./backups-restic/README.md)
 
 [addons-docs]: (https://docs.kubermatic.com/kubeone/v1.10/guides/addons/)
-[backups-addon]: (./backups-restic)
-[restic]: (https://restic.net/)
```

Comment:
Oh well, actually those links should looks like this

```markdown
[addons-docs]: https://docs.kubermatic.com/kubeone/v1.10/guides/addons/
[backups-addon]: ./backups-restic/README.md
[restic]: https://restic.net/
```

We mistakenly added parenthesis around where they are not belong.